Bug Fix: Contact Heading Centering Issue

shredtechular opened the pull request:
This PR includes a fix for the heading centering issue described in #29 .

## Issue Problem
The problem for this issue is stemming from a class that includes a negative margin that was added to the div above it.  In general, negative margins are not recommended and tend to cause other issues, like the centering issue described in the issue.

## Issue Solution
As described above, the simplest solution that I can think of is to remove the negative margin class from the div above the issue.  

I'm not sure if a negative margin is actually required here?  I actually think the spacing is more consistent and better without the negative margin?  Screen caps below of before fix (with the negative margin) and after the fix (with the negative margin).  If I am mistaken and the spacing should match as it does before the fix, please let me know and we can figure out a different way of tackling this issue.

### Before the fix (current production):
<kbd>
<img width="1164" alt="screen shot 2017-10-08 at 3 25 57 pm" src="https://user-images.githubusercontent.com/2396774/31320228-fa18f15e-ac3e-11e7-95f1-74e638136ec0.png">
</kbd>

### Building the fix included in this PR it would look like this:
<kbd>
<img width="1186" alt="screen shot 2017-10-08 at 3 25 28 pm" src="https://user-images.githubusercontent.com/2396774/31320230-028603c2-ac3f-11e7-8302-01dc4b740b6d.png">
</kbd>

**UPDATE**
Per feedback, pushed 9ef1eb0 that fixed email addresses on this page as well changing "seakhealing" to "seekhealing".  This commit fixes #28.

howtohumanbetter commented:
Beautiful. Please note that there are also typos in all these email addresses: each one should be @seekhealing.org -- NOT "seakhealing"

shredtechular commented:
A-ha!  Now I see what you mean with the incorrect emails!  OK, I've fixed that now in 9ef1eb0 and I've updated this PR description to include fixing #28 as well.  Below screen capture showing the correct emails.  Luckily the links were all "seekhealing" and not "seakhealing" though.

<img width="1181" alt="screen shot 2017-10-11 at 11 51 00 pm" src="https://user-images.githubusercontent.com/2396774/31478553-218389c8-aedf-11e7-9d1d-ddbe2b9a61b2.png">

shredtechular commented:
@j00bar this PR is ready to be reviewed now.
